vyper/builtins/functions.py

```diff
@@ -1479,8 +1479,8 @@ def evaluate(self, node):
         value, shift = [i.value for i in node.args]
         if value < 0 or value >= 2 ** 256:
             raise InvalidLiteral("Value out of range for uint256", node.args[0])
-        if shift < -(2 ** 127) or shift >= 2 ** 127:
-            raise InvalidLiteral("Value out of range for int128", node.args[1])
+        if shift < -256 or shift > 256:
```

Comment:
i think we could add a comment, that this validation is to prevent the compiler from hanging rather than for correctness (because the post-folded constant would get validated anyways).